.github/workflows/tests.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        tests: [unit-tests]
+        tests: [unit-tests, unit-tests-race]
```

Comment:
do we need both? The logic is still tested if the `-race` flag is enabled so maybe just the second would be enough?

Reply:
good point, will remove